configuration/wireless/access-point.md

```diff
@@ -70,10 +55,10 @@ Type or copy the following information into the dnsmasq configuration file and s
 
 ```
 interface=wlan0      # Use the require wireless interface - usually wlan0
-  dhcp-range=192.168.0.2,192.168.0.20,255.255.255.0,24h
+  dhcp-range=192.168.4.2,192.168.4.20,255.255.255.0,24h
 ```
```

Comment:
Finally got a chance to look at this. Just out of interest, why did you change this to .4?

Reply:
I changed it because many routers use 192.168.0.x/24 or 192.168.1.x/24 for the localnet. The AP subnet must be different.

Reply:
Note that the IP address of the wlan interface was also changed to 192.168.4.1/24 in /etc/dhcpcd.conf. The subnets must match in these two locations.